app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,78 @@
+class FoodsController < ApplicationController
+  before_action :set_food, only: %i[show edit update destroy]
+  before_action :authenticate_user! # Add this line if you want to authenticate users
+
+  # GET /foods or /foods.json
+  def index
+    # @foods = Food.all
+    @foods = current_user.food.includes(:recipe_foods)
+  end
```

Comment:
- [ ] Please consider removing `commented codes` as it will make your codebase messy, Remember if you ever like to see the codes you have written once you can use your commit history.